src/sensors.c

```diff
@@ -64,9 +64,7 @@ static char *sensor_type_name_map[] =
 	"temperature",
 # define SENSOR_TYPE_POWER       3
 	"power",
-# define SENSOR_TYPE_CURRENT     4
-	"current",
-# define SENSOR_TYPE_UNKNOWN     5
```

Comment:
I didn't check the complete "#if" part.
This code is only for API below 0x400 and this does not have this feature as I see in other comments. So I have to clean "sensors_labeltypes_t".